README.md

```diff
@@ -1,2 +1,67 @@
 # VR-AA-replacer
 Lighten your VR controller by moving the AA batteries elsewhere
+
+## Overview
+
+This project provides a 3D-printable replacement part that allows you to remove heavy AA batteries from VR controllers and relocate them to a more convenient position. The main objectives are to:
+
+- **Minimize total weight** by replacing AA batteries with lighter AAA batteries
+- **Improve center of mass balance** by relocating batteries away from the controller head
+- **Enhance comfort** during extended VR sessions by reducing controller weight and improving weight distribution
+
+## The Replacement Part
+
+![AA Replacer Part](AA%20replacer.png)
+
+The replacement part is designed to fit in place of AA batteries in VR controllers. It features:
+
+- Lightweight 3D-printable design
+- Two countersunk M3 screw holes for electrical contacts
+- Compact form factor that matches AA battery dimensions
+
+## Wiring Instructions
+
+The replacement part uses **two countersunk M3 screws** as electrical contacts to provide power to your VR controller:
+
+1. **Install M3 screws**: Insert countersunk M3 screws into the designated holes on the replacement part
+2. **Connect wires**: The M3 screws serve as clamping points for wires that connect to your external battery pack
+3. **Wire routing**: Run wires from the screws to your relocated battery pack (AAA batteries in a remote location)
+4. **Secure connections**: The screws can be tightened to clamp the wires directly, ensuring reliable electrical contact
+
+### Wiring Setup
+- **Positive contact**: Connect the positive wire from your battery pack to one M3 screw
+- **Negative contact**: Connect the negative wire from your battery pack to the other M3 screw
+- **Wire gauge**: Use appropriate wire gauge for the current requirements of your VR controller
+- **Strain relief**: Ensure wires have adequate strain relief to prevent disconnection during use
+
+## Example: VR Table Tennis Paddle
+
+![VR TT Paddle Example](VR%20TT%20paddle.png)
+
+The image above demonstrates a practical application where:
+
+- **Original setup**: Heavy AA batteries located in the paddle head
+- **Improved setup**: Lighter AAA batteries relocated to the handle
+- **Benefits achieved**:
+  - **Reduced total weight**: AAA batteries are lighter than AA batteries
+  - **Better balance**: Moving batteries to the handle shifts the center of mass closer to your grip
+  - **Improved gameplay**: Better weight distribution enhances control and reduces fatigue
+
+This example shows how the AA-replacer part enables you to transform the weight distribution of VR accessories, making them more comfortable and natural to use.
+
+## Files Included
+
+- `AA replacement.step` - 3D CAD file for the replacement part
+- `AA replacer.png` - Image of the replacement part design
+- `VR TT paddle.png` - Example application showing before/after battery placement
+
+## Usage Tips
```

Comment:
Change this.
You can print it in any material, one shell should be enough if you chase minimum weight.